app/views/admin/schedules/show.html.haml

```diff
@@ -11,8 +11,15 @@
   .row
     .col-md-2
       Selected schedule
-      = check_box_tag @conference.short_title, @schedule.id, (@schedule.id == @selected_schedule.try(:id)),
-        method: :patch, url: (admin_conference_program_path(@conference.short_title) + '?[program][selected_schedule_id]='),
+      :ruby
+        if @schedule.track
+          value = @schedule == @schedule.track.selected_schedule
+          url = update_selected_schedule_admin_conference_program_track_path(@conference.short_title, @schedule.track) + '?[track][selected_schedule_id]='
+        else
+          value = @schedule.id == @selected_schedule.try(:id)
```

Comment:
I don't know, I don't think that we need it. It's there because I just copy-pasted the expression from the existing code. In the expression I wrote (https://github.com/openSUSE/osem/pull/1631#discussion-diff-134726302R16) I don't use the id, but compare the two objects directly.
@Ana06 can you help as with this?